File: server-demo/src/main/java/com/example/HelloServiceImpl.java
package com.example;

import java.util.Date;

import com.alibaba.dubbo.config.annotation.Service;

@Service(version = "1.0.0")
public class HelloServiceImpl implements HelloService {

	@Override
    public String sayHello(String name) {
        return "Hello, " + name + ", " + new Date();
    }

}

File: kotlin-server-demo/src/main/kotlin/com/example/HelloServiceImpl.kt
package com.example;

import java.util.Date;

import com.alibaba.dubbo.config.annotation.Service;

@Service(version = "1.0.0")
class HelloServiceImpl : HelloService {

    override fun sayHello(name: String): String {
        return "Hello, " + name + ", " + Date()
    }

}